CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changes to PostCSS Preset Env
 
+### 6.6.0 (February 28, 2019)
+
+- Moved browserslist caching to before each processing.
```

Comment:
Do you think it'd make sense to explicitly point out the effect of this change (performance improvement; reduce overhead per file) here? The current line wouldn't make a lot of sense to me if I wasn't already familiar with the internals.

Reply:
@kblomster, agreed. Let me revisit this comment.